docs/api/native-image.md

```diff
@@ -273,15 +273,15 @@ Returns `NativeImage` - The cropped image.
   * `width` Integer (optional) - Defaults to the image's width.
   * `height` Integer (optional) - Defaults to the image's height.
   * `quality` String (optional) - The desired quality of the resize image.
-    Possible values are `good`, `better` or `best`. The default is `best`.
+    Possible values are `good`, `better`, or `best`. The default is `best`.
```

Comment:
We can make this `String` be typed as an enum if we write "Can be X, Y or Z"

```suggestion
    Can be `good`, `better` or `best`. The default is `best`.
```

Reply:
@Zearin i'll approve once the above comment has been addressed ☝️